pre_commit/main.py

```diff
@@ -85,6 +85,10 @@ def _add_run_options(parser: argparse.ArgumentParser) -> None:
         '--all-files', '-a', action='store_true', default=False,
         help='Run on all the files in the repo.',
     )
+    mutex_group.add_argument(
+        '--fail-fast', action='store_true', default=False,
```

Comment:
`store_true` implies `default=False`

Reply:
Ok, I will change it. My change was based on the previous entry, which is a `store_true` and also has `default=False`.